spec/epochs_spec.lua

```diff
@@ -794,8 +797,11 @@ describe("epochs", function()
 
 			local rewardsForDelegatesThatLeft = 12500000000
 
-			-- check the epoch was updated
-			local distributions = epochs.getEpoch(epochIndex).distributions
+			-- check the epoch was properly distributed
+			if not distributedEpoch then
+				error("Distributed epoch is nil")
```

Comment:
Why not just assert not nil? Just wondering if there's a logical difference here.

Reply:
linting - it's not smart enough to recognize assert.is_not_nil passing means the object is not nil